bgpd/bgp_vty.c

```diff
@@ -3037,6 +3037,30 @@ DEFUN (no_bgp_default_show_hostname,
 	return CMD_SUCCESS;
 }
 
+/* BGP Extended Message */
+DEFUN (bgp_ext_message,
+       bgp_ext_message_cmd,
+       "bgp extended-message",
+       "BGP specific commands\n"
+       "Enable BGP Extended Message (65535 bytes)\n")
+{
+	VTY_DECLVAR_CONTEXT(bgp, bgp);
+	bgp->extended_message = BGP_EXTENDED_MESSAGE_ENABLED;
+	return CMD_SUCCESS;
+}
+
+DEFUN (no_bgp_ext_message,
+       no_bgp_ext_message_cmd,
+       "no bgp extended-message",
+       NO_STR
+       "BGP specific commands\n"
+       "Enable BGP Extended Message (65535 bytes)\n")
+{
+	VTY_DECLVAR_CONTEXT(bgp, bgp);
+	bgp->extended_message = BGP_EXTENDED_MESSAGE_DISABLED;
+	return CMD_SUCCESS;
```

Comment:
When this CLI was disabled after BGP sessions has come up with established? To ensure this CLI takes an affect we need to clear the BGP sessions?

Reply:
Yes, I’ll update that. Thanks.